src/synology_dsm/api/virtual_machine_manager/guest.py

```diff
@@ -0,0 +1,90 @@
+"""DSM Virtual Machine data."""
+from synology_dsm.helpers import SynoFormatHelper
+
+
+class SynoVirtualMachineManager:
+    """Class containing Virtual Machine Guests"""
+
+    API_KEY = "SYNO.Virtualization.API.Guest"
+    ACTION_API_KEY = "SYNO.Virtualization.API.Guest.Action"
+
+    def __init__(self, dsm):
+        """Constructor method."""
+        self._dsm = dsm
+        self._data = {}
+
+    def update(self):
+        """Updates Virtual Machine Guest data."""
+        raw_data = self._dsm.get(self.API_KEY, "list")
+        if raw_data:
+            self._data = raw_data["data"]
```

Comment:
you should create an own class for the guest object (_suggest `SynoVmmGuest`_) which holds all the data and exposes properties (_compare with surveillance_station_)